jerry-core/include/jerryscript-core.h

```diff
@@ -786,14 +799,43 @@ typedef enum
                                            *   value: value for rejecting */
 } jerry_promise_event_type_t;
 
+/**
+ * Filter types for jerry_promise_set_callback callback function.
+ * The callback is only called for those events which are enabled by the filters.
+ */
+typedef enum
+{
+  JERRY_PROMISE_FILTER_DISABLE = 0, /**< disable reporting of all events */
```

Comment:
Since the enum is called promise_event_filter_t, I'd rather use jerry_promise_event_filter_disable. I think that better represents that the filter is applied on the events not on the promises.

Reply:
Then all enums should be renamed to event filter, but I think that is too long.

Reply:
Indeed, that's exactly what I meant. I think the extra `_EVENT` 6 characters does not add to much, but makes it more consistent. The longest option is currently `JERRY_PROMISE_FILTER_ASYNC_REACTION_JOB` which is 39 characters which would be 45, which still less then the half of the current column limit. So I'd still support adding the `_EVENT` into the enum values.